lib/src/stripe_payment.dart

```diff
@@ -174,6 +174,10 @@ class StripePayment {
     final result = await _channel.invokeMethod('confirmSetupIntent', intent.toJson());
     return SetupIntentResult.fromJson(result);
   }
+
+  static openApplePaySetup() async {
```

Comment:
can you add some information why and where this is needed?

Reply:
I need open ApplePaySetup in somecase and StripePaymentPlugin.m and TPSStripeManager.m implemented this. but I can't call this method and I add openApplePaySetup method in stripe_payment.dart file.

https://github.com/jonasbark/flutter_stripe_payment/blob/master/ios/Classes/StripePaymentPlugin.m#L65
https://github.com/jonasbark/flutter_stripe_payment/blob/master/ios/Classes/TPSStripeManager.m#L877